azurerm_function_app/variables.tf

```diff
@@ -15,17 +15,8 @@ variable "resource_name_prefix" {
   description = "The prefix used to name all resources created."
```

Comment:
There are few things that need to be fixed that are not directly visible from this page:
* Remove all commented/unused code
* I'd like to discuss the usage of the variable function. Was it used for debug? Can we remove it?